src/main/scala/mesosphere/marathon/core/task/tracker/impl/steps/PostToEventStreamStepImpl.scala

```diff
@@ -38,22 +38,34 @@ class PostToEventStreamStepImpl @Inject() (
       case TASK_ERROR | TASK_FAILED | TASK_FINISHED | TASK_KILLED | TASK_LOST =>
         // Remove from our internal list
         maybeTask match {
-          case Some(task) => postEvent(appId, status, task)
-          case None       => log.warn(s"Task not found. Do not post event for '{}'", taskId.getValue)
+          case Some(task) =>
+            postEvent(timestamp, appId, status, task)
+          case None =>
+            log.warn(s"Received terminal task event but task not found. " +
```

Comment:
There is no substitution on this first line – no need to use String interpolation.
I've talked with our _Onsite English Expert_ and came up with a neater (and shorter) message:
`log.warn(s"Received TASK_RUNNING for unknown task [${taskId}] of [$appId]. Not posting event.")`

Btw: the `taskId` is only used for logging, so you can as well initialize it with `val taskId = status.getTaskId.getValue` – then the messages fit in one line. Yay ;)